airgun/views/virtwho_configure.py

```diff
@@ -111,6 +112,7 @@ class VirtwhoConfiguresView(BaseLoggedInView, SearchableViewMixin):
             'Actions': ActionsDropdown("./div[contains(@class, 'btn-group')]"),
         },
     )
+    dialog = Pf4ConfirmationDialog()
```

Comment:
This has been added to `BaseLoggedInView` as a widget already, so you don't actually need to add it. Doesn't hurt though. 

https://github.com/SatelliteQE/airgun/pull/681

Reply:
Thank you very much mshriver, BaseLoggedInView has been deleted from VirtwhoConfiguresView.